src/ape/api/accounts.py

```diff
@@ -72,6 +72,14 @@ def alias(self) -> Optional[str]:
         """
         return None
 
+    # TODO: In 0.9, make this an @abstactmetod
+    @property
+    @raises_not_implemented
```

Comment:
Might I suggest instead that this returns optional, and then None by default? so that account plugins can "opt-in" to sharing their public key (they might not have this property, e.g. Safe account or expose access to it e.g. Ledger)